### PR TITLE
[NEW] Voice Channel Client

### PR DESCRIPTION
**Pull request by hrahul2605**
## Proposed changes (including videos or screenshots)

This PR includes changes regarding voice channels client

### CODE REVIEW

**Comment by hrahul2605** [comment]
@dougfabris , made this separate PR for client changes

**Comment by hrahul2605** [comment]
Closing this, continued [here](https://github.com/hrahul2605/Rocket.Chat/pull/1)

File: client/views/room/Room/VoiceRoom.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+			device: {},
+			produce: true,
+			consume: true,
+			displayName: room.u.name || 'Anonymous',
+			peerID: room.u._id,
+		});
+
+		try {
+			await roomClient.join();
+			roomClient.on('newConsumer', (consumer: types.Consumer, peer) => {
```
Comment (dougfabris) [inline client/views/room/Room/VoiceRoom.tsx:32]: I don't think we should manipulate the DOM directly, we could replace it with React's approach

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: client/views/room/Room/VoiceRoom.tsx (hunk 1 above)
Comment (hrahul2605) [reply client/views/room/Room/VoiceRoom.tsx:32]: Yes, I did this for testing purpose and to check the connectivity.
Will be creating the Peer/Avatar component replacing this.

Comment (dougfabris) [reply client/views/room/Room/VoiceRoom.tsx:32]: I got your point, let's move like this for now!

**Review by dougfabris — APPROVED** [review]

**Pull request merged**